plugins/scaffolder-backend/src/lib/files/serializeDirectoryContents.ts

```diff
@@ -44,19 +44,36 @@ export async function serializeDirectoryContents(
     dot: true,
     gitignore: options?.gitignore,
     followSymbolicLinks: false,
+    // In order to pick up 'broken' symlinks, we oxymoronically request files AND folders yet we filter out folders
+    // This is because broken symlinks aren't classed as files so we need to glob everything
+    onlyFiles: false,
     objectMode: true,
     stats: true,
   });
 
   const limiter = limiterFactory(10);
 
   return Promise.all(
-    paths.map(async ({ path, stats }) => ({
+    paths
+      .filter(({ dirent }) => !dirent.isDirectory())
+      .filter(({ dirent, path }) => {
+        if (!dirent.isSymbolicLink()) return true
+        if (!fs.existsSync(joinPath(sourcePath, path))) return true  // We only want symlinks that DO NOT exist
```

Comment:
could possibly check that the symlink is within the work dir and reject if it's outside, but since we have the existence check it's not a big deal probably